.github/workflows/tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out the repo
+      uses: actions/checkout@v2
+
+    - name: Set up Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+
+    - name: Install dependencies with Yarn
+      run: yarn
+
+    - name: Run unit tests
+      run: yarn test
+
+  browser-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out the repo
+      uses: actions/checkout@v2
+
+    - name: Set up Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+
+    - name: Set up Ruby for Jekyll
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: '2.6'
+
+    - name: Install dependencies with Yarn
+      run: yarn && yarn --cwd docs/
+
+    - name: Start local Design System website server
+      run: |
+        yarn --cwd docs/ start &
+        sleep 5
+
+    - name: Run browser tests
+      env:
+        SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
+        SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
```

Comment:
I added these secrets to this repo's [settings page](https://github.com/cfpb/design-system/settings/secrets). The access key is tied to my CFPB sauce labs account.

Reply:
Can we leave a single line comment (should be supported in yml with a hash) that says this?